### ISSUE TITLE
Python debug configurations can no longer be added

### ISSUE THREAD

**Issue by bwateratmsft** [microsoft/vscode-python-debugger#203]
I have Python extension version v2024.0.1 and Python Debugger extension version v2024.0.0. I am no longer able to add Python debug configurations, such as the Python Attach configuration.

This is gone when doing Add Configuration:
![image](https://github.com/microsoft/vscode-python-debugger/assets/36966225/480b4145-c0ee-4736-bbac-11a0cc69b34b)

Original bug here: https://github.com/microsoft/vscode-docker/issues/4232

**Comment by paulacamargo25** [comment]
Thanks for your issue report @bwateratmsft, the activation of the extension depends on the files you have, try opening a Python file and then return to launch.json, the Python Debugger option still does not appear?

**Comment by paulacamargo25** [comment]
Can you also check if the Python Debugger extension is installed?

**Comment by bwateratmsft** [comment]
Yes, the Python Debugger extension is installed. After opening a Python file, the option reappears. Previously this wasn't necessary--is the activation criteria different for the Python Debugger extension?

**Comment by paulacamargo25** [comment]
Yes, it is a little bit different, we are only using the activation events for debug cases, but i think we can add more activation events. I will change the issue to feature request.

**Comment by bwateratmsft** [comment]
Had this issue come up again
